scripts/operations/validation/ncn1.3.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
What purpose is this script serving? It isn't linked from the documentation that I can see.